src/scss/components/_ratings.scss

```diff
@@ -1,30 +1,37 @@
-$star-count: 6;
-$star-size-small: 16px;
-$star-size-medium: 28px;
-$star-size-large: 42px;
-$star-spacing: $star-count;
+$star-count             : 6;
+$star-size-extraSmall   : 12px;
+$star-size-small        : 16px;
+$star-size-medium       : 28px;
+$star-size-large        : 42px;
+$star-spacing           : $star-count;
 
 .c-rating {
     @extend %c-icon--star--empty !optional;
     background-repeat: repeat-x;
     background-size: contain;
     display: inline-block;
     height: $star-size-small;
-    width: $star-size-small * $star-count + $star-spacing;
+    width: ($star-size-small * $star-count) + $star-spacing;
+}
+
+.c-rating--extraSmall {
```

Comment:
Can you add this modifier into the ratings page on the docs site too

Reply:
done - PR is open and linked in the description :)